Signal/src/ViewControllers/MenuActionsViewController.swift

```diff
@@ -288,29 +283,58 @@ class MenuActionSheetView: UIView, MenuActionViewDelegate {
         actionStackView.spacing = CGHairlineWidth()
 
         actions = []
+        actionViews = []
+        hapticFeedback = HapticFeedback()
 
         super.init(frame: frame)
 
         backgroundColor = UIColor.ows_light10
         addSubview(actionStackView)
-        actionStackView.ows_autoPinToSuperviewEdges()
```

Comment:
Do we eventually want to purge our usage of `ows_autoPinToSuperviewEdges` entirely?  Is there a card for this?

Reply:
I'd like to get rid of it entirely, but I consider it really low priority. I think with the name change (adding the `ows_` prefix) we'll naturally phase it out.